packages/twenty-server/src/engine/workspace-manager/workspace-migration-runner/workspace-migration-runner.service.ts

```diff
@@ -507,4 +516,27 @@ export class WorkspaceMigrationRunnerService {
       COMMENT ON FOREIGN TABLE "${schemaName}"."${name}" IS '@graphql({"primary_key_columns": ["id"], "totalCount": {"enabled": true}})';
     `);
   }
+
+  private async alterForeignTable(
+    queryRunner: QueryRunner,
+    schemaName: string,
+    name: string,
+    alterations: WorkspaceMigrationAlterForeignTableAlteration[] | undefined,
+  ) {
+    const alterationsQuery = alterations
+      ?.map((alteration) => {
+        if (alteration.action === WorkspaceMigrationColumnActionType.DROP) {
```

Comment:
actions that don't match will return undefined in the array and will then be converted to "undefined" string with the join.
You can filter it out before the join 👍 
Also maybe use a switch for better visibility?
```typescript
    const alterationsQuery = alterations
      ?.map((alteration) => {
        switch (alteration.action) {
          case WorkspaceMigrationColumnActionType.DROP:
            return `DROP COLUMN "${alteration.columnName}"`;
          case WorkspaceMigrationColumnActionType.CREATE:
            return `ADD COLUMN "${alteration.columnName}" ${alteration.columnType}`;
          default:
            return '';
        }
      })
      .filter(Boolean)
      .join(', ');
```

Reply:
merci coco!